### DISCUSSION TITLE
Notify not enable

### DISCUSSION THREAD

**Discussion by rahulnrt** [WalletConnect/walletconnect-monorepo#3952] [Developer Support]
Dear team,
I am not able to enable notify API in my Explorer app

**Comment by glitch-txs** [comment]
could you elaborate? what error are you getting?

**Comment by rahulnrt** [comment]
[image: image.png]
find the above screenshot here I am not able to enable the Notify API
option can you please assist? I am not able to enable push notification
service in our application

**Reply by glitch-txs** [reply]
I can't see the image

**Comment by rahulnrt** [comment]
I want to enable notify api option in our walletconnect modal explorer
I want to enable the push notification in our dapp

can you arrange a call for me : 8358977907
